deploy/packages/external-libraries.yaml

```diff
@@ -31,7 +31,6 @@ packages:
       - gmsh+metis~mpi+oce+openmp+shared@4.4.1
       - hdf5~mpi@1.10.5
       - intel-mkl@2019.3.199
-      - jdk@1.8.0_131-b11
```

Comment:
no longer used?

Reply:
Upstream dependency. I don't think anyone uses JDK by itself…

Reply:
We should enable `hpcviewer` and that needs JDK.

Reply:
Fine.